neural_speed/core/ne_layers.c

```diff
@@ -9203,7 +9242,31 @@ static void ne_compute_forward_rope_f16(const struct ne_compute_params* params,
 
         float theta = freq_scale * (float)p;
 
-        if (!is_neox) {
+        if (is_longrope) {
+          float scale_factor = 1.1902380714238083;
```

Comment:
this value is from paper or could also get from config ？

Reply:
get from max_seq_len